frontend/smart-kickers-game/src/components/Heatmap/Heatmap.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import HeatMap from 'react-heatmap-grid';
+import './Heatmap.css';
+import { Colors } from './Colors';
+
+function Heatmap({ heatmap }) {
+  const heatmapDim = heatmap.length;
+  const array = new Array(heatmapDim).fill(0).map(() => '');
+  let numbersCopy = JSON.parse(JSON.stringify(heatmap));
+
+  for (let i = 0; i < heatmapDim; i++)
+    for (let j = 0; j <= i; j++) {
+      numbersCopy[i][j] = numbersCopy[i][j] + numbersCopy[j][i] - numbersCopy[j][i];
+      numbersCopy[j][i] = numbersCopy[i][j];
+    }
```

Comment:
Could you explain why do do such operations here, what's the point of them because I don't really understand.

Reply:
It was a mistake. It has been changed.